app/assets/javascripts/update.js

```diff
@@ -41,6 +42,7 @@ function editBody(){
       }
 
     event.preventDefault();
+    this.blur();
```

Comment:
There definitely seem to be some duplication in this code. How could you DRY it up?